Automattic/jetpack
# Reorg: Split out .gitignore

Splitting out a separate .gitignore for the jetpack plugin project.

#### Changes proposed in this Pull Request:
* see above.

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Confirm there are no gross changes with what has been removed from the repo root .gitignore.
* I tested ^ by running yarn install, yarn build-jetpack, and `composer install` within `projects/packages/connection` as a validation that I didn't regress anything with the packages.

#### Proposed changelog entry for your changes:
* n/a
